.github/workflows/release.yml

```
@@ -17,3 +17,33 @@ jobs:
+
+            **Changelog:**
+
+            ${{ github.event.release.body }}
+        if: github.event.release.prerelease == false
+
+      - name: Actions for Discord (Preview)
+        uses: Ilshidur/action-discord@0.4.0
+        env:
+          DISCORD_WEBHOOK: ${{ secrets.TAILSCALE_DISCORD_WEBHOOK }}
+        with:
+          args: |
```

actionlint (1.7.7)

property "version" is not defined in object type {}

(expression)